### ISSUE TITLE
UiSnackbar need new callback onQueueEnd

### ISSUE THREAD

**Issue by projct1** [JosephusPaye/Keen-UI#252]
Now i have to use this crutch:
```javascript
onHide() {
    if (vm.$refs.snackbarContainer.$children.length === 1) {
        //do-something
    }
},
```

**Comment by JosephusPaye** [comment]
Hi @rorc,

Good idea, will add this to the next version (as an event on UiSnackbarContainer).

**Comment by rorc** [comment]
@JosephusPaye Cool, thanks 😄

**Comment by JosephusPaye** [comment]
Released in [v1.1.0](https://github.com/JosephusPaye/Keen-UI/releases/tag/v1.1.0).
